packages/host/app/components/operator-mode/container.gts

```diff
@@ -110,7 +115,11 @@ export default class OperatorModeContainer extends Component<Signature> {
     registerDestructor(this, () => {
       delete (globalThis as any)._CARDSTACK_CARD_SEARCH;
       this.operatorModeStateService.clearStacks();
+      this.commandService.unregisterCommandHandler(this);
     });
+    this.commandService.registerCommandHandler(this, 'patch', (args) =>
+      this.patchCard(args),
```

Comment:
this should really be a ember concurrency task that is performed. The interesting this about ember concurrency tasks is that from the outside the can look synchronous, since it is ok to call `perform()`  in a synchronous manner. that actually might be a interesting way to solve the event handler synchronicity issue. from within the component you can see the asynchronicity and await loading state.